src/_prebid.jsx

```diff
@@ -62,10 +82,6 @@ export default class PrebidContainer extends Component {
     if (!window.pbjs) {
       window.pbjs = { que: [this.configure] };
       window.pbjs__slots = [];
-      window.pbjsAdServerTimeout = setTimeout(() => {
-        console.log('[prebid] global timeout');
-        this.sendAdserverRequest();
-      }, this.props.adserverTimeout);
```

Comment:
`bidsBackHandler` is called when `bidderTimeout` is threw?

Reply:
http://prebid.org/dev-docs/publisher-api-reference.html#pbjsrequestbidsrequestobj
>Callback to execute when all the bid responses are back or the timeout hits.